SafeExamBrowser.UserInterface.Mobile/Windows/BrowserWindow.xaml.cs

```diff
@@ -445,5 +447,20 @@ private void LoadText()
 			FindMenuText.Text = text.Get(TextKey.BrowserWindow_FindMenuItem);
 			ZoomText.Text = text.Get(TextKey.BrowserWindow_ZoomMenuItem);
 		}
+
+		public void FocusToolbar(bool forward)
+		{
+			throw new NotImplementedException();
```

Comment:
Unfortunately, as long as we still need to maintain two UIs you'd need to implement the functionality from the desktop UI here as well.

Reply:
I was working under the assumption that all my work is not needed for the mobile part. Not sure if I got this information from you or Anton.

If that's wrong, how can I test the mobile parts? Is that a Xamarin project?

Reply:
Aha, okay, yes that might very well be correct, sorry. I must admit I don't even know what they agreed upon, if you could check back with Anton and if it's the case, then that's certainly perfectly fine, thanks.

Reply:
> If that's wrong, how can I test the mobile parts? Is that a Xamarin project?

No, it's just a bit a different implementation of the same UI with bigger buttons etc. so that it can be used easier on touch devices. You can enable it in the configuration under "Browser View Mode" -> "Touch Optimized": https://safeexambrowser.org/windows/win_usermanual_en.html#AppearancePane